doc/us-en/setup/Install-OpenEdge-on-Linux.md

```diff
@@ -1,2 +1,161 @@
+
+### Install Docker
+
+> OpenEdge offers two startup modes. To start using ***docker*** container mode (recommended), you need to complete the docker installation first.
+
+**Notice:**
+
++ The official Dockerfile is offered for multi-stage builds. If you need to build the relevant image yourself, The version of docker you installed should be above 17.05.
++ The production environment can run the image using a lower version of Docker, which is currently tested to a minimum usable version of 12.0.
++ According to the [Official Release Log](https://docs.docker.com/engine/release-notes/#18092), the version of docker version < 18.09.2 has some security implications. It is recommended to install/update the Docker version to 18.09. 2 and above.
+
+Can be installed by the following command(Suitable for linux-like systems, [Supported Platforms](./Support-platforms.md)):
+
+```shell
+curl -sSL https://get.docker.com | sh
+```
+
+#### Ubuntu
+
+Command:
+
+```shell
+sudo snap install docker # Ubuntu16.04 after
+sudo apt install docker.io # Ubuntu 16.04 before
+```
+
+#### CentOS
+
+Command:
+
+```shell
+yum install docker
+```
+
+***Notice***: 
+
++ After the Docker installation is complete, use the following command to view the installed Docker version.
+
+```shell
+docker version
+```
+
+#### Debian 9/Raspberry Pi 3
+
+Command:
+
+```shell
+curl -sSL https://get.docker.com | sh
+```
+
+**For more details, please see the [official documentation](https://docs.docker.com/install/).**
+
+### Install Python2.7 and Python runtime dependency package
+
+> + OpenEdge provides Python Runtime, which supports running code written in Python version 2.7. If you plan to start using ***native*** process mode, you need to install Python 2.7 and the package you depend on. If you plan to start in ***docker*** container mode, you do not need to perform the following steps.
+
+#### Ubuntu 18.04 LTS/Debian 9/Raspberry Pi 3
+
+Commands:
+
+```shell
+sudo apt update
+sudo apt upgrade
+sudo apt install python2.7
+sudo apt install python-pip
+sudo pip install protobuf grpcio
+```
+
+#### CentOs 
+
+Execute the following command to check the installed Python version:
+
+```shell
+python -V
+```
+
+If the result is not installed, you can install it using the following command:
+
+```shell
+yum install python
+yum install python-pip
+yum install protobuf grpcio
+```
+
+Or install from source code:
+
+```shell
+yum install gcc openssl-devel bzip2-devel
+wget https://www.python.org/ftp/python/2.7.15/Python-2.7.15.tgz
+tar xzf Python-2.7.15.tgz
+make altinstall
+curl "https://bootstrap.pypa.io/get-pip.py" -o "get-pip.py"
+python2.7 get-pip.py
+pip install protobuf grpcio
+```
+
+> Enter the command `python -V` to see that the Python version is 2.7.* and the installation is correct.
+
+### Specify the default Python version
+
+In some cases, you need to specify the default Python version for the above installed version. Complete with the following command (Valid after reboot):
```

Comment:
In some cases, you need to specify the default version for Python of the above installed. Specifically, you can execute the commond as below(Valid after reboot).